### ISSUE TITLE
card browser spinner's UI is confusing

### ISSUE THREAD

**Issue by LCB2021029-Badri** [ankidroid/Anki-Android#13172]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by LCB2021029-Badri** [comment]
@vvs @Ramblurr @jberkel @the-kenny I would like to resolve this issue right away, can I have the permissions for this ?

**Comment by LCB2021029-Badri** [comment]
![Screenshot_1675155193](https://user-images.githubusercontent.com/96579549/215713358-a479d6fc-eaeb-4575-9f52-115e85d9fc15.png)
I would like to improvise the UI here for better experience.

**Comment by david-allison** [comment]
Hi! Thanks for your enthusiasm and helping out with open source!

You haven't filled out the issue template, you've pinged a ton of people who I don't believe are currently contributing to the project, and you haven't explained what you're planning on doing.

I do feel the browser needs work UI-wise, but I'd much rather we encouraged work on current open issues before defining a new UI direction, especially as a 'first contribution'. Once you've got a few issues under your belt, then we're in a much less risky position where we can look into accepting and starting to work through larger UI changes.

Closing as WONTFIX for now, ping me once the above is resolved and I'll reopen and we can get this started.

https://github.com/ankidroid/Anki-Android/wiki/User-Interface-contribution
